build/CFNetwork/dll/CFNetwork.vcxproj

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)\..\common\winobjc.nuproj.common.props" Condition="Exists('$(SolutionDir)\..\common\winobjc.nuproj.common.props')" />
```

Comment:
Nevermind

---
In reply to: [105063164](https://github.com/Microsoft/WinObjC/pull/2189#discussion_r105063164) [](ancestors = 105063164)